aws-greengrass-testing-platform/aws-greengrass-testing-platform-api/src/main/java/com/aws/greengrass/testing/platform/UnixCommands.java

```diff
@@ -58,10 +58,12 @@ public UnixCommands(final Device device, final PillboxContext pillboxContext) {
     public byte[] execute(CommandInput input) throws CommandExecutionException {
         final StringJoiner joiner = new StringJoiner(" ").add(input.line());
         Optional.ofNullable(input.args()).ifPresent(args -> args.forEach(joiner::add));
+        //TODO: For windows, we need to format the path to be platform agnostic, but not for the config. We also need
+        // test cases for OTF to make sure the compatibility when crossing platforms.
```

Comment:
The formatting is only needed for path originating for windows-unix cross-platform compatibility, thus it should not be placed in the execute command which further formats the configs. The configs are required for the local deployment.

We remove the formatting right now to unblock the local deployment with configs, but will add it back when it only format the path and the OTF have corresponding test cases in the future.